### PR TITLE
11573 Misc fixes

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#11573

*Description of changes:*
- changed disabled checkboxes to blue
- updated whitespace around button panel
- updated BN rules
- fixed delivery address incomplete code
- prepared for GP (ie, partners)
- fixed typo
- display proprietor/partners in Doc Delivery section
- fixed unit test helpers so errors display in the tests themselves
- app version = 3.5.4

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/assets/styles/overrides.scss [hunk 1]
```diff
@@ -120,7 +120,7 @@
     opacity: 0.4;
   }
   .v-input--selection-controls__input i {
-    color: $gray7 !important;
+    color: $app-blue !important;
```
Comment (severinbeauvais) [inline src/assets/styles/overrides.scss:123]: Per Tracey.

File: src/mixins/people-roles-mixin.ts [hunk 2]
```diff
@@ -86,6 +86,11 @@ export default class PeopleRolesMixin extends Vue {
     return this.orgPersonList.filter(person => this.isProprietor(person))
   }
 
+  /** The list of partners. */
+  get partners (): OrgPersonIF[] {
+    return this.orgPersonList.filter(person => this.isPartner(person))
+  }
```
Comment (severinbeauvais) [inline src/mixins/people-roles-mixin.ts:92]: Prep for GP, although there is some question about whether to differentiate Partners from Proprietors. I think they're the same, except SPs can have only 1 and GPs need at least 2.

Anyway, this is Future Severin's problem :)

File: tests/unit/AddEditOrgPerson.spec.ts [hunk 3]
```diff
@@ -128,9 +128,9 @@ const emptyPerson = { ...EmptyOrgPerson }
  * @returns the value of the last named event for the wrapper.
  */
 function getLastEvent (wrapper: Wrapper<AddEditOrgPerson>, name: string): any {
-  const eventsList: Array<any> = wrapper.emitted(name)
-  const events: Array<any> = eventsList[eventsList.length - 1]
-  return events[0]
+  const eventsList = wrapper.emitted(name)
+  const events = eventsList && eventsList[eventsList.length - 1]
+  return events && events[0]
```
Comment (severinbeauvais) [inline tests/unit/AddEditOrgPerson.spec.ts:133]: Previously, an error with an emitted event could report an error on line 132. Now, this function is safe, and the error will be reported in the unit test itself (ie, with the correct line number displayed).

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Pull request merged**